src/retrieve-data-helpers/fhir-metadata-retrieval.js

```diff
@@ -13,20 +13,41 @@ function retrieveFhirMetadata(testUrl) {
     let testFhirServer = testUrl;
     if (!testFhirServer) {
       const parsed = queryString.parse(window.location.search);
-      testFhirServer = parsed.fhirServiceUrl || store.getState().fhirServerState.defaultFhirServer;
+      testFhirServer = parsed.fhirServiceUrl ||
+        localStorage.getItem('PERSISTED_fhirServer') ||
+        store.getState().fhirServerState.defaultFhirServer;
     }
-    const headers = { Accept: 'application/json+fhir' };
+    const headers = { Accept: 'application/json' };
```

Comment:
The correct FHIR MIME type is `application/fhir+json` (STU3+), though it used to be `application/json+fhir` (pre-STU3). Did you change this to just `application/json` because you ran into a FHIR server that didn't like the previous value we were using here?

Reply:
Yes, if the HSPC default fhir server's secured endpoint (DSTU2) is hit with `application/json+fhir`, we get a 406 Not Acceptable error, rather than the expected 401. Ideally, we could read from the conformance statement to find the appropriate MIME type to use.

@travistcummings is a 406 error expected if we hit `https://api.hspconsortium.org/cdshooksdstu2/data/Patient`?